src/AppInstallerRepositoryCore/Public/AppInstallerRepositorySource.h

```diff
@@ -14,6 +14,17 @@
 
 namespace AppInstaller::Repository
 {
+    using namespace std::string_view_literals;
+
+    constexpr std::string_view s_Source_WingetCommunityDefault_Name = "winget"sv;
+    constexpr std::string_view s_Source_WingetCommunityDefault_Arg = "https://winget.azureedge.net/cache"sv;
+    constexpr std::string_view s_Source_WingetCommunityDefault_Data = "Microsoft.Winget.Source_8wekyb3d8bbwe"sv;
+
+    constexpr std::string_view s_Source_WingetMSStoreDefault_Name = "msstore"sv;
+    // TODO: update to use prod endpoint
+    constexpr std::string_view s_Source_WingetMSStoreDefault_Arg = "https://winget-test.azureedge.net/msstore"sv;
+    constexpr std::string_view s_Source_WingetMSStoreDefault_Data = "Microsoft.Winget.MSStore.Source_8wekyb3d8bbwe"sv;

```

Comment:
This should not need to be in the header.

Reply:
The only reason I move them in the header is s_Source_WingetCommunityDefault_Name is used in AggregatedSource.cpp

Do you want me to move them all back to cpp and create a copy in AggregatedSource.cpp Or keep the *_Name constants here and move *_Arg and *_Data back to cpp? 

---
In reply to: [467209878](https://github.com/microsoft/winget-cli/pull/527#discussion_r467209878) [](ancestors = 467209878)

Reply:
Move them back to the cpp and don't make a copy elsewhere because they shouldn't be needed.

---
In reply to: [467338302](https://github.com/Microsoft/pkg/pull/527#discussion_r467338302) [](ancestors = 467338302,467209878)